List metrics that relate to Pod security admission (#41814)
kubernetes/website: content/en/docs/concepts/security/pod-security-admission.md

- Valid updates to `.spec.activeDeadlineSeconds`
- Valid updates to `.spec.tolerations`

> ## Metrics
> You can look at the following Prometheus metrics exposed by kube-apiserver.

Reviewer: ```suggestion
## Metrics

Here are the Prometheus metrics exposed by kube-apiserver:
```

> - `pod_security_errors_total`: metric contains the number of errors preventing normal evaluation. Non-fatal errors may result in the latest restricted profile being used for evaluation.

Reviewer: ```suggestion
- `pod_security_errors_total`: This metric indicates the number of errors preventing normal evaluation.
  Non-fatal errors may result in the latest restricted profile being used for evaluation.
```

> - `pod_security_evaluations_total`: metric contains the number of policy evaluations that occurred, not counting ignored or exempt requests.
>   during exporting.

Reviewer: ```suggestion
- `pod_security_evaluations_total`: This metric indicates the number of policy evaluations that have occurred,
  not counting ignored or exempt requests during exporting.
```

> - `pod_security_exemptions_total`: metric contains the number of exempt requests, not counting ignored or out of scope requests.

Reviewer: ```suggestion
- `pod_security_exemptions_total`: This metric indicates the number of exempt requests, not counting ignored
  or out of scope requests.
```
